tss/.htaccess

```diff
@@ -0,0 +1,28 @@
+# RDF Time Series Snippets (TSS)
+RewriteEngine on
+DirectorySlash Off
+
+## TSS Ontology content negotiation resources
+RewriteCond %{HTTP_ACCEPT} text/html
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.html#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} application/ld\+json
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.jsonld#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} application/n-triples
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.nt#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.rdf#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} text/turtle
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.ttl#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} .+
+
+RewriteRule (.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/406.html [NE,L,R=406]
+
+# === TSS ===
+RewriteCond %{REQUEST_URI} tss
+RewriteRule tss/?$ https://knowledgeonwebscale.github.io/rdf-timerseriessnippets/tss.ttl [NE,R,L]
+RewriteCond %{REQUEST_URI} tss/spec
+RewriteRule tss/spec/?$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets [NE,R,L]
+RewriteCond %{REQUEST_URI} (about|from|until|pointType|points|timePath|valuePath|Snippet)$
+RewriteRule ^(.*)$ https://%{SERVER_NAME}/rdf-timeseriessnippets/$1.tss.conneg [NE,R,L]
```

Comment:
What is the expectation of SERVER_NAME here?  I think that's this server?
Why use RewriteCond vs putting patterns in RewriteRule directly?

Reply:
`SERVER_NAME` is auto-replaced by Apache with `http://w3id.org/`.
This makes writing rules easier to test locally.
I copied from [rml](https://github.com/perma-id/w3id.org/tree/master/rml) to allow for more complex redirections in the future.